jinahub/encoders/audio/VGGISHAudioEncoder/tests/requirements.txt

```diff
@@ -1,2 +1,3 @@
-git+https://github.com/jina-ai/jina.git@v2.0.0#egg=jina[hub]
-pytest==6.2.4
\ No newline at end of file
+jina~=2.0
```

Comment:
```suggestion
```

Reply:
We need this. It is in every tests/requirements.txt

Reply:
We agreed to remove it, and in testing CI script you can see that jina gets installed before the requirements

Reply:
So should I remove it from all tests/requirements.txt ?

Reply:
Yes

Reply:
This still needs to be removed